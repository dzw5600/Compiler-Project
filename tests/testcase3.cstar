// Testing switch statements and for loops (TODO):
// 1. Switch statement
// 2. Break keyword optional
// 3. Strings and chars
// Variable incrementing

int i = 0;

while (i < 5)
{
    switch (i)
    {
        case (0):
            print('0'); // char
        case (1):
            print("one"); // string
            break;
        default:
            break;
    }

    set i = i + 1;
}

print("All done!");
