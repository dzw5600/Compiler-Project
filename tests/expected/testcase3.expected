0
one
All done!